standards/SPIFFE_Workload_API.md

```diff
@@ -360,6 +366,122 @@ When interfacing with legacy JWT validators, the `FetchJWTBundles` method can be
 
 The `FetchJWTBundles` method returns bundles keyed by the SPIFFE ID of the trust domain. When validating a JWT-SVID, the validator MUST use the bundle corresponding to the trust domain of the subject. If a JWT bundle for the specified trust domain is not present, then the token is untrusted.
 
+## 7. WIT-SVID Profile
+
+The WIT-SVID Profile of the SPIFFE Workload API provides a set of gRPC methods which can be used by workloads to retrieve [WIT-SVIDs](WIT-SVID.md) and their related trust bundles. This profile outlines the signature of these methods, as well as related client and server behavior.
+
+Unlike the X.509-SVID and JWT-SVID profiles, implementation of the WIT-SVID profile is optional. Where a server does not support the WIT-SVID profile, calls to all RPCs defined by the WIT-SVID profile MUST return the "Unimplemented" gRPC status code. If the client receives an "Unimplemented" status code calling a WIT-SVID profile RPC, it SHOULD NOT attempt to reconnect to the server using the WIT-SVID profile.
+
+### 7.1 Profile Definition
+
+The WIT-SVID Profile RPCs and associated messages are defined below. For the complete Workload API service definition, see [workloadapi.proto](workloadapi.proto).
+
+```protobuf
+service SpiffeWorkloadAPI {
+  /////////////////////////////////////////////////////////////////////////
+  // WIT-SVID Profile
+  /////////////////////////////////////////////////////////////////////////
+
+  // Fetch WIT-SVIDs for all SPIFFE identities the workload is entitled to,
+  // as well as related information like trust bundles. As this information
```

Comment:
> as well as related information like trust bundles

There is a similar comment on the `WITSVID` message, but I don't see where a bundle is delivered as part of this response. I don't think we need one anyways due to the asymmetric nature of WIT auth ... I checked the JWT-SVID profile and don't see one there either.

Reply:
👍  Yeah - I think in my original draft I had this returning the trust bundle but after discussion we removed that - looks like I forgot to update this comment.

Reply:
Addressed in 62c51f3